docs/app/Examples/elements/Segment/Groups/SegmentHorizontalSegmentsExample.js

```diff
@@ -4,11 +4,11 @@ import { Segment } from 'stardust'
 export default class SegmentHorizontalSegmentsExample extends Component {
   render() {
     return (
-      <Segment.Segments className='horizontal'>
+      <Segment.Group aligned='horizontal'>
```

Comment:
Should 'vertical' be supported here as well, or inferred as the default, 
by the omission of 'horizontal' ?

> Levi Thomason <mailto:notifications@github.com>
> 26 June 2016 at 22:39
>
> In 
> docs/app/Examples/elements/Segment/Groups/SegmentHorizontalSegmentsExample.js 
> <https://github.com/TechnologyAdvice/stardust/pull/287#discussion_r68510275>:
>
> > @@ -4,11 +4,11 @@ import { Segment } from 'stardust'
> > export default class SegmentHorizontalSegmentsExample extends 
> Component {
> > render() {
> > return (
> > - <Segment.Segments className='horizontal'>
> > + <Segment.Group aligned='horizontal'>
>
> Since the |horizontal| 
> <http://semantic-ui.com/elements/segment.html#horizontal-segments> 
> class here applies to the layout of the segments and there is only one 
> option, I think we should use a boolean prop:
>
> <Segment.Group horizontal />
>
> image 
> <https://cloud.githubusercontent.com/assets/5067638/16365125/b53c0eae-3bab-11e6-8a78-00810e7c74bb.png>
>
> Please let me know if I'm over looking anything.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/TechnologyAdvice/stardust/pull/287/files/a628afbe7bc8b86e4cc139252db0d32941a9e279#r68510275>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AAWHsy5NzDjxjcY32cSo1-PmIc6bqqo6ks5qPvGogaJpZM4I-lPR>.
>

-- 
Sent from Postbox 
<https://www.postbox-inc.com/?utm_source=email&utm_medium=siglink&utm_campaign=reach>